restcomm/restcomm.rvd/src/main/webapp/languages/es.json

```diff
@@ -69,7 +70,7 @@
 
 	"designerSettingsModalSettingLabel": "Configuración",
 	"designerSettingsModalRestcommAPILabel": "Restcomm API",
-	"designerSettingsModalCredencialsLabel1": "Credenciales de acceso a la Restcomm API (Debe proporcionarlos si los hay)",
+	"designerSettingsModalCredencialsLabel1": "Credenciales para accesar a la Restcomm API. Debe proporcionarlos si",

```

Comment:
Grammar error found. Change for: 
"designerSettingsModalCredencialsLabel1": "Credenciales para acceder a la RestComm API. Debe proporcionarlos si",